string topic_name
string publisher_ip
int32 publisher_port
string subscriber_ip
string subscriber_node_name
int64 total_bytes
